ONSdigital/response-operations-ui
# Migrate Rops from GCR to GAR

# What and why?
We are in the process of migrating our repos from GCR to GAR, this PR provides the migration for Rops
# How to test?
Check image location is correct https://console.cloud.google.com/artifacts/docker/ons-ci-rmrasbs/europe-west2/images/response-operations-ui?project=ons-ci-rmrasbs

Build out environment and deploy a new version of rops through spinnaker or locally through helm:
```
docker image tag - pr-957
config branch - migrate-rops-from-gcr-to-gar
```

You will need to update the helm chart image location either by doing a helm deploy locally or updating the helm chart in GCP with the correct image location (europe-west2-docker.pkg.dev/ons-ci-rmrasbs/images), otherwise the deployment will fail. https://console.cloud.google.com/kubernetes/deployment/europe-west2/k8s-ras/robinm3/response-operations-ui/yaml/view?project=ras-rm-dev
# Jira
https://jira.ons.gov.uk/browse/RAS-1232
